scripts/reactor_deployment.py

```diff
@@ -8,6 +8,72 @@
 
 
 # # # # # # # # # # # # Constituent Functions # # # # # # # # # # #
+def pull_start_index(df, start_year):
+    """
+    This function takes in a dataframe and a start year and returns the
+    index of the start year.
+
+    Parameters
+    ----------
+    df : :class:`pandas.DataFrame`
+        The dataframe to pull the start year from.
+    start_year : int
+        The year to start the deployment of advanced reactors.
+
+    Returns
+    -------
+    start_index : int
+        The index of the start year.
+    """
+    start_index = df[df['Year'] == start_year].index[0]
+
+    return start_index
+
+
+def capacity_increase(df, base_col, rate, start_year, end_year):
+    """
+    This function takes in an increase rate, and creates a new column in the
+    dataframe populated with the increased capacity.
+
+    Parameters
+    ----------
+    df : :class:`pandas.DataFrame`
+        The dataframe to add the new column to.
+    base_col : str
+        The name of the column to use as the base capacity.
+        This column should contain the net capacity.
+    rate : float
+        The percentage of change in capacity year-to-year
+        (e.g. 1.01 for a 1% increase).
+    start_year : int
+        The year to start the increase.
+    end_year : int
+        The year to end the increase.
+
+    Returns
+    -------
+    df : :class:`pandas.DataFrame`
+        The dataframe with the new column added.
+    """
+    # Apply lambda function for the range from start_year to end_year
```

Comment:
Is this inline comment helping your comprehension or just restating the command below? It seems like the latter, to me.
```suggestion
```

Reply:
I see your point, especially for function calls. I frequently use inline comments because of my previous frustrations figuring out what different bits and function calls were being used for as you've noticed. 

I have recently gone back and added docstrings to many of the functions (which did not exist before) in this repo so it is less of a pressing need (hopefully), but it's still habit that I like because it explicitly lays out the flow.